README.md

```diff
@@ -1,2 +1,62 @@
+| nes_vaccination_record/vaccination_management/medication/manufacturer | Vaccine manufacturer |
+| nes_vaccination_record/vaccination_management/medication/batch_id | Vaccine batch code |
+| nes_vaccination_record/vaccination_management/medication/expiry | Vaccine expiration data. *Note*: we don't record this (VMT might have it) |
+| nes_vaccination_record/vaccination_management/medication_supply_amount/amount_description | Amount given. *Note*: NCDS don't record this yet, probably will in the future |
+| nes_vaccination_record/vaccination_management/reason:0 | Map to the FHIR Immunization.statusReason element when ISM is ‘Vaccine dose not given’ and to ‘Immunization.reasonCode’ when ISM is ‘Vaccine dose administered’. *Suggestion*: we might want to record the SNOMED code for reason also? |
+| nes_vaccination_record/vaccination_management/administration_details/route | FHIR code - https://www.hl7.org/fhir/valueset-immunization-route.html (NCDS have a longer list) |
+| nes_vaccination_record/vaccination_management/administration_details/body_site | FHIR code - https://www.hl7.org/fhir/valueset-immunization-site.html (NCDS have a longer list) |
+| nes_vaccination_record/vaccination_management/nes_vaccination_event_details/vaccination_administration_purpose|code | Either booster dose (at0009) or primary (at0008) |
+| nes_vaccination_record/vaccination_management/dose_number" | Dose number |
+| nes_vaccination_record/vaccination_management/note | *Note*: supported but may not be used in NCDS |
+| nes_vaccination_record/vaccination_management/appointment_id | 14ebf56b-8d76-439b-8bfe-884e1e8fa569 |
+| nes_vaccination_record/vaccination_management/appointment_id\|issuer | TODO REMOVE? Issuer |
+| nes_vaccination_record/vaccination_management/appointment_id\|assigner | TODO REMOVE? Assigner |
+| nes_vaccination_record/vaccination_management/appointment_id\|type | TODO REMOVE? Prescription |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/target_disease:0 | Disease SNOMED code |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|code | Country code |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|value | Country value |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|terminology | Country ISO |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/vaccination_centre | Vaccination centre name *Question*: overlap with health care facility at top? |
+| nes_vaccination_record/vaccination_management/time | Time composition created |
+
+## How to use the composition
+
+We can create a new composition in EHRbase using the below cURL command or our [postman collection](https://nesdigital.atlassian.net/wiki/spaces/NDSENG/pages/130383926). Note that the address of the endpoint differs from the canonical JSON endpoint.
```

Comment:
suggestion: Which instance of EHRBase should I have used to test this out? I went down the route of running it locally, but there were quite a few extra steps involved prior to being able to create the composition, such as creating the EHR and template. Do we need to add some more info to this part or have i missed something due to my beginner level knowledge of EHRbase?